examples/inlinekeyboard2.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#Simple inline keyboard bot with multiple callback Query Handler.
+"""
+This Bot uses the Updater class to handle the bot.
+First, a few callback functions are defined as callback query handler. Then, those functions are passed to
+the Dispatcher and registered at their respective places.
+Then, the bot is started and runs until we press Ctrl-C on the command line.
+Usage:
+Example of a bot that uses inline keyboard that has multiple callbacks Query Handler.
+Send /start to initiate the conversation.
+Press Ctrl-C on the command line to stop the bot.
+"""
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup
+from telegram.ext import Updater, CommandHandler, CallbackQueryHandler, ConversationHandler
+import logging
+
+# Enable logging
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                    level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+FIRST, SECOND = range(2)
+
+def start(bot, update):
+    user = update.message.from_user
+    logger.info("User %s started the conversation." % user.first_name)
+    keyboard = [
+        [InlineKeyboardButton(u"Next", callback_data=str(FIRST))]
+    ]
+    reply_markup = InlineKeyboardMarkup(keyboard)
+    update.message.reply_text(
+        u"Start handler, Press next",
```

Comment:
Do we mark strings in the other examples as unicode?

Reply:
No! We are not marking strings in other example as unicode, but I think it's a good practice to mark strings as unicode.

Reply:
Should I keep it or make this example symmetrical with other examples?